### PR TITLE
ci: Workaround ci_extended container setup gid issue

### PR DESCRIPTION
**Pull request by asdil12**
https://progress.opensuse.org/issues/93880

This is only for testing and is to be removed before merge:

```diff
-on: push
+on: [push, pull_request]
```

### CODE REVIEW

File: .github/workflows/ci_extended.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 name: ci extended
 # yamllint disable-line rule:truthy
-on: push
+on: [push, pull_request]
```
Comment (okurz) [inline .github/workflows/ci_extended.yml:4]: I assume you do that just for testing. Please revert before merging

**Review by okurz — CHANGES_REQUESTED** [review]

File: .github/workflows/ci_extended.yml (hunk 1 above)
Comment (asdil12) [reply .github/workflows/ci_extended.yml:4]: yes - as mentioned in the first comment.

File: container/os-autoinst_dev/Dockerfile [hunk 2]
```diff
@@ -5,7 +5,7 @@ FROM opensuse/tumbleweed
 ENV NAME openQA test environment
 ENV LANG en_US.UTF-8
 
-RUN zypper ar -f -G 'http://download.opensuse.org/repositories/devel:/openQA/openSUSE_Tumbleweed' devel_openqa
+RUN zypper ar -f -G 'http://download.opensuse.org/repositories/devel:/openQA/openSUSE_Tumbleweed' devel_openqa && zypper -n in system-user-qemu && zypper clean
```
Comment (okurz) [inline container/os-autoinst_dev/Dockerfile:8]: Please add a comment here to explain the workaround

Comment (asdil12) [reply container/os-autoinst_dev/Dockerfile:9]: done

**Review by Martchus — APPROVED** [review]

**Review by okurz — APPROVED** [review]

**Pull request merged**